### PR TITLE
introduce localization to menu tab

### PR DESCRIPTION
**Pull request by oniram88**

### CODE REVIEW

File: lib/alchemy/solidus/engine.rb [hunk 1]
```diff
@@ -12,6 +12,15 @@ class Engine < ::Rails::Engine
         ]
       end
 
+      initializer 'alchemy_solidus.i18n' do |app|
+        Alchemy::Solidus::Engine.instance_eval do
+            
+          files = Dir[File.join(File.dirname(__FILE__), '../..', 'alchemy_solidus_*.yml')]
+          ::I18n.load_path.concat(files)
```
Comment (tvdeyen) [inline lib/alchemy/solidus/engine.rb]: You don't need that. Rails will pick this up for us

**Comment by tvdeyen** [comment]
Thanks. The initializer is not needed. Could you please remove it?

**Comment by oniram88** [comment]
ok, removed the initializer

**Comment by tvdeyen** [comment]
Thanks

**Pull request merged**